src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/CSharpSyntaxFormattingOptions.cs

```diff
@@ -0,0 +1,222 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Formatting;
+
+namespace Microsoft.CodeAnalysis.CSharp.Formatting
+{
+    [Flags]
+    internal enum SpacePlacement
+    {
+        IgnoreAroundVariableDeclaration = 1,
```

Comment:
```suggestion
        IgnoreAroundVariableDeclaration = 1 << 0,
```

up to you :)   i like the symettry.